   Reason: Typecast error with -pedantic
   Source: Jonathan Dahan <jedahan@gmail.com>
 Upstream: No
--- darkice-0.19/src/Util.cpp
+++ darkice-0.19/src/Util.cpp
@@ -308,7 +308,7 @@ Util :: fileAddDate ( const char * str,
     strftime( strdate, 128, format, localtime (&now));
 
     // search for the part before the extension of the file name
-    if ( !(last = strrchr( str, '.')) ) {
+    if ( !(last = strrchr( (char *) str, '.')) ) {
         last = (char *) str + strlen( str);
     }
